host-configs/lc-builds/blueos/xl_2020_X.cmake

```diff
@@ -7,10 +7,10 @@
 
 set(RAJA_COMPILER "RAJA_COMPILER_XLC" CACHE STRING "")
 
-set(CMAKE_CXX_FLAGS_RELEASE "-O3 -qxlcompatmacros -qlanglvl=extended0x -qalias=noansi -qsmp=omp -qhot -qnoeh -qsuppress=1500-029 -qsuppress=1500-036" CACHE STRING "")
-set(CMAKE_CXX_FLAGS_RELWITHDEBINFO "-O2 -g -qxlcompatmacros -qlanglvl=extended0x -qalias=noansi -qsmp=omp -qhot -qnoeh -qsuppress=1500-029 -qsuppress=1500-036" CACHE STRING "")
-set(CMAKE_CXX_FLAGS_DEBUG "-O0 -g -qsmp=omp:noopt " CACHE STRING "")
-set(CMAKE_EXE_LINKER_FLAGS "-Wl,-z,muldefs" CACHE STRING "")
+set(CMAKE_CXX_FLAGS_RELEASE "-O3 -std=c++14 -std=c++11 -qxlcompatmacros -qlanglvl=extended0x -qalias=noansi -qsmp=omp -qhot -qsuppress=1500-029 -qsuppress=1500-036" CACHE STRING "")
+set(CMAKE_CXX_FLAGS_RELWITHDEBINFO "-O2 -g -std=c++14 -std=c++11 -qxlcompatmacros -qlanglvl=extended0x -qalias=noansi -qsmp=omp -qhot -qsuppress=1500-029 -qsuppress=1500-036" CACHE STRING "")
+set(CMAKE_CXX_FLAGS_DEBUG "-O0 -g -std=c++14 -std=c++11 -qsmp=omp:noopt " CACHE STRING "")
```

Comment:
why both?

Reply:
BLT_CXX_STD is set in the top level RAJA/CMakeLists.txt. For the duplicate `-std=c++11` flag, I was trying to make sure our scripts matched exactly what was in Gitlab. Let me try removing it from both our scripts and Gitlab.

Reply:
For some reason, doing `set(BLT_CXX_STD c++14 CACHE STRING "")` only adds `-std=c++1y` to the compilation line. Let's leave `-std=c++14` in for now.